modules/compliance-operator-install.adoc

```diff
@@ -8,12 +8,14 @@
 [role="_abstract"]
 Install the Compliance Operator using Operator Hub.
 
+////
```

Comment:
why comment out instead of just removing?

Reply:
Yes, good question.
I was thinking of keeping the module in case we had to retrieve it for some reason.